src/pages/index.tsx

```diff
@@ -0,0 +1,77 @@
+import cn from "classnames";
+import Button from "@rescui/button";
+import * as styles from "../components/MainPage/MainPage.module.css";
+import Layout from "../components/Layout";
+import googleLogoSvg from "../components/MainPage/companies/google.svg";
+import jbLogoSvg from  "../components/MainPage/companies/jb.svg";
+import {Link} from "gatsby";
+
+const SPONSORS = [
+    { name: 'Google', image: googleLogoSvg },
+    { name: 'JetBrains', image: jbLogoSvg },
+]
+
+export default function MainPage() {
+    return <Layout
+        intro={
```

Comment:
The Intro block is relevant only for the main page, why not pass it as a child to Layout?
Now, in my opinion, the layout contains logic that might not be there.